front/app/containers/IdeasShowPage/index.tsx

```diff
@@ -97,19 +90,22 @@ const IdeasShowPage = () => {
     return <PageNotFound />;
   }
 
-  if (idea) {
+  if (idea && project) {
+    const isIdeaInCurrentPhase =
+      idea.data.relationships.phases.data.filter(
+        (phase) => phase.id === participationContext?.id
+      ).length > 0;
+    const showCTABar =
+      isIdeaInCurrentPhase &&
+      participationContext?.attributes.participation_method === 'voting';
+    const showCTABarAtTopOfPage = !isSmallerThanTablet && showCTABar;
+
     return (
-      <VotingContext projectId={project?.data.id}>
+      <VotingContext projectId={project.data.id}>
         <Box
           background={colors.white}
-          mt={
-            // Only move the content down the page if the idea is in a current voting phase
-            isSmallerThanTablet ||
-            participationContext?.attributes.participation_method !==
-              'voting' ||
-            !isIdeaInCurrentPhase
-              ? undefined
-              : `${stylingConsts.menuHeight.toString()}px`
+          pt={
+            showCTABarAtTopOfPage ? `${stylingConsts.menuHeight}px` : undefined
           }
```

Comment:
I found this hard to understand. It was not clear we intend to add margin-top because we show the CTA bar. I've instead created variables that describe this.